typescript/nomad-sdk/src/nomad/govern/index.ts

```diff
@@ -57,77 +94,88 @@ export class CallBatch {
   }
 
   // Build a governance transaction from this callbatch
-  async build(
-    overrides?: ethers.Overrides,
-  ): Promise<ethers.PopulatedTransaction> {
+  async build(): Promise<ethers.PopulatedTransaction> {
     if (this.built) return this.built;
-
     const [domains, remoteCalls] = utils.associateRemotes(this.remote);
-
     this.built =
-      await this.core.governanceRouter.populateTransaction.executeGovernanceActions(
+      await this.governorCore.governanceRouter.populateTransaction.executeGovernanceActions(
         this.local,
         domains,
         remoteCalls,
       );
-
     return this.built;
   }
 
+  // Return the batch hash for the specified domain
+  domainHash(domain: number): string {
```

Comment:
let's rename to `batchHash(domain:number)` -- it should be enough that it's implied to be for the domain by the parameter